pkg/xds/generator/system_names/api.go

```diff
@@ -0,0 +1,5 @@
+package system_names
```

Comment:
maybe the whole `admin_generator` should be moved to a separate package, so it's not in general "generator" package? and then move this to that package? WDYT?